mypy.ini

```diff
@@ -58,6 +58,9 @@ ignore_missing_imports = True
 [mypy-torch.for_onnx.onnx]
 ignore_missing_imports = True
 
+[mypy-torch.ao.quantization.experimental.observer]
```

Comment:
Is it okay for me to modify mypy.ini? This rule just applies to my oberver.py file

Reply:
I think this file should be excluded from being tracked by git, its fine to modify it, it shouldn't be reflected here.